libsolidity/interface/StandardCompiler.cpp

```diff
@@ -557,8 +558,11 @@ boost::variant<StandardCompiler::InputsAndSettings, Json::Value> StandardCompile
 		ret.evmVersion = *version;
 	}
 
-	if (settings.isMember("remappings") && !settings["remappings"].isArray())
-		return formatFatalError("JSONError", "\"settings.remappings\" must be an array of strings.");
+	if (settings.isMember("remappings"))
```

Comment:
Any reason for changing this? It's still the same and now has redundant brackets...

Reply:
Thanks, that was introduced during the rebase.